src/AzureAuth/CommandMain.cs

```diff
@@ -23,6 +23,7 @@ namespace Microsoft.Authentication.AzureAuth
     /// The command main class parses commands and dispatches to the corresponding methods.
     /// </summary>
     [Command(Name = "azureauth", Description = "A CLI interface to MSAL authentication")]
+    [Subcommand(typeof(CommandInfo))]

```

Comment:
<curious> - is this the first time we are ever introducing a subcommand?

Reply:
Yes, it is. The usage of Subcommand is from Lasso documentation.